packages/dashboard/src/components/Recommendation/Visualization/ImageVisualization.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+import { getImageVisualizationStyles } from './ImageVisualization.styles';
+import { IImageVisualizationProps, IImageVisualizationStyles } from './ImageVisualization.types';
+
+export const ImageVisualization: React.SFC<IImageVisualizationProps> = (props: IImageVisualizationProps) => {
+  const getClassNames = classNamesFunction<{}, IImageVisualizationStyles>();
+  const classNames = getClassNames(getImageVisualizationStyles);
+  const { imageSrc, imageAlt } = props;
+
+  if (imageSrc) {
+    return (
+      <div className={classNames.imageIllustrationContainerStyle}>
+        <img src={imageSrc} alt={imageAlt} className={classNames.imageIllustrationStyle} />
```

Comment:
> <img src={imageSrc} alt={imageAlt} className={classNames.imageIllustrationStyle} /> [](start = 7, length = 84)

do you want aria-hidden? We have used this in case we find the image is just for aesthetic look there is no point reading the image

Reply:
Good point, done.